index.js

```diff
@@ -76,9 +76,13 @@ program
 
 if ((os.platform() === 'win32' && require('is-administrator')()) || process.getuid() <= 0) {
     async.waterfall([(cb) => {
-        return properties.parse(config.CONF_FILE, {
-            path: true
-        }, cb);
+        if (fs.existsSync(config.CONF_FILE)) {
```

Comment:
since you are already doing some other async process, it's probably better to just use the async version of `fs.exists` rather than the sync version

Reply:
actually ya, since you are using an `async.waterfall` here, you could just break this into a `fs.exists` step, then proceeding to `properties.parse` step.

In fact why not just convert this to a `async.seq` I think your code will read much better this way.

Reply:
waterfall has one of the difference which is well used here is that it does  not execute the rest of the function if something fails but returns the error instead. I think this functionality is useful here. The other difference is that the async.seq takes the initial parameter as the main callback argument. This  I think will require to change  how the script is written here. But to achieve better readability, I think I should change give name to all those anonymous methods in the array and pass the names only.

Reply:
@vilyapilya, I agree with @mikehu on using `fs.exists` - but [`fs.exists`](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback) has already been deprecated - instead, you can use `fs.access`

Reply:
And fs.exists has a different callback signature.

Reply:
I mean, you can use `fs.access` in the following way, if you want:
```
        return fs.access(config.CONF_FILE, (error) => {
            if (error) {
                return cb(null, null);
            }
            return properties.parse(config.CONF_FILE, {
                path: true
            }, cb);
        });
```
instead of
```
        if (fs.existsSync(config.CONF_FILE)) {
            properties.parse(config.CONF_FILE, {
                path: true
            }, cb);
        } else {
            cb(null, null);
        }
```

But it will be needed to be tested, if you add this change.

Reply:
replaced